src/System.Device.Gpio/Interop/Unix/libgpiod/Interop.libgpiod.cs

```diff
@@ -5,54 +5,55 @@
 using System;
 using System.Device.Gpio;
 using System.Runtime.InteropServices;
+using NativeLong = System.IntPtr;
```

Comment:
I'd add a comment before this, something like

```C#
// Since this is only used on Linux, and in C on Linux sizeof(long) == sizeof(void*) this is a valid alias.